idb/postgres/postgres_migrations.go

```diff
@@ -665,7 +667,7 @@ func processAccountTransactions(txnrows <-chan idb.TxnRow, addressStr string, ad
 			result.assetHolding[txn.AssetID] = updateCreate(result.assetHolding[txn.AssetID], txn.Round)
 		}
 
-		if accounting.AssetOptOutTxn(stxn) {
+		if accounting.AssetOptOutTxn(stxn) && stxn.Txn.Sender == address {
```

Comment:
Migration accounting bug 2, we were marking all accounts referenced in asset closes as deleted = true instead of just the sender.